t/selenium/Work_Editor.html

```diff
@@ -43,6 +43,16 @@
 	<td>name=edit-work.languages.0</td>
 	<td>label=[No lyrics]</td>
 </tr>
+<tr>
+	<td>assertEval</td>
+	<td>!!window.document.getElementById('add-language')</td>
```

Comment:
This is the only unusual addition to Selenium tests. 
I would prefer using `assertElementNotPresent` instead.
This command has to be added to `t/selenium.js` though.

Reply:
I tried adding this (implemented in the same way `assertElementPresent` is), but the web driver kept returning an element despite it not existing on the page, and I couldn't figure out why. This method works fine, at least.